bindings/python/iota_sdk/types/common.py

```diff
@@ -7,7 +7,8 @@
 HexStr = NewType("HexStr", str)
 
 HD_WALLET_TYPE = 44
-HARDEN_MASK = 1 << 31;
+HARDEN_MASK = 1 << 31
```

Comment:
Are these still used since we have Bip44?

Reply:
No

Reply:
@qrayven Please remove

Reply:
removed

Reply:
Does it need a changelog entry?

Reply:
removed for nodejs bindings too, changlog updated for both